docs/getting_started.md

```diff
@@ -24,14 +26,63 @@ async def produce():
         await asyncio.sleep(5)
 
 
-async def main():
+async def start():
     await stream_engine.start()
     await produce()
+
+
+async def shutdown():
     await stream_engine.stop()
 
 
 if __name__ == "__main__":
-    asyncio.run(main())
+    loop = asyncio.get_event_loop()
+    try:
+        loop.run_until_complete(start())
+        loop.run_forever()
+    finally:
+        loop.run_until_complete(shutdown())
+        loop.close()
+```
+
+*(This script is complete, it should run "as is")*
+
+In the previous example you can see some boiler plate regarding how to start the program. We recommend to use [aiorun](https://github.com/cjrh/aiorun),
```

Comment:
Maybe add a title here `# Production deployment `

Reply:
I don't think this should be titled as `Production deployment `, this is `THE recommended usage` regardless the environment (only when a web server is not present). If we named this `Production deployment `, we are inferring that during the development phase another way to run the code can be used (which is true), but it will imply a code change when going to production which might cause issues. I think we should encourage consistency and the same way of run the `engine` should be used in all environments, at least for now.

We could add more emphasis on `# The recommended usage` instead.

Reply:
Yes, makes sense.